mflowgen/full_chip/construct.py

```diff
@@ -25,32 +25,33 @@ def construct():
   parameters = {
     'construct_path'    : __file__,
     'design_name'       : 'GarnetSOC_pad_frame',
-    'clock_period'      : 100.0,
+    'clock_period'      : 2.0,
     'adk'               : adk_name,
     'adk_view'          : adk_view,
     # Synthesis
-    'flatten_effort'    : 3,
+    'flatten_effort'    : 1,
     'topographical'     : True,
     # RTL Generation
     'array_width'       : 32,
     'array_height'      : 16,
     'interconnect_only' : False,
     # Include Garnet?
-    'soc_only'          : False,
+    'soc_only'          : True,
     # SRAM macros
-    'num_words'      : 2048,
-    'word_size'      : 64,
-    'mux_size'       : 8,
-    'corner'         : "tt0p8v25c",
-    'partial_write'  : True,
+    'num_words'         : 2048,
+    'word_size'         : 64,
+    'mux_size'          : 8,
+    'corner'            : "tt0p8v25c",
+    'partial_write'     : True,
     # Low Effort flow
-    'express_flow' : False,
-    'skip_verify_connectivity' : True,
-    'lvs_hcells_file' : 'inputs/adk/hcells.inc',
+    'express_flow'      : False,
+    'skip_verify_connectivity'  : True,
+    'lvs_hcells_file'   : 'inputs/adk/hcells.inc',
     'lvs_connect_names' : '"VDD VSS VDDPST"',
     # DRC rule deck
-    'drc_rule_deck' : 'calibre-drc-chip.rule',
-    'antenna_drc_rule_deck' : 'calibre-drc-antenna.rule'
+    'drc_rule_deck'     : 'calibre-drc-chip.rule',
+    'antenna_drc_rule_deck'     : 'calibre-drc-antenna.rule',
+    'nthreads'          : 4
```

Comment:
I don't think we want this for the full chip flow

Reply:
Agreed!